topics/ui/controls/link.md

```diff
@@ -10,127 +10,266 @@
+        <img src="link_when_not_to_1_correct.png" width="378" alt="The Commit Message input with primary and secondary actions as buttons."/>
+    </td>
+    <td>
+        <format color="E55765" style="bold">Incorrect</format>
+        <img src="link_when_not_to_1_incorrect.png" width="378" alt="The Commit Message input with primary and secondary actions as links."/>
+    </td>
+  </tr>
+</table>
 
-A link can be a part of a checkbox or radio button label or of any stand-alone text, like the text in an [empty state](empty_state.md) or in [context help](context_help.md).
+### Simple UI
 
-![](09_part_of_text_1.png){width=403}
+Use a [button](button.topic) or a [drop down list](drop_down.md) when a UI is not constrained.
+These controls can be selected from the keyboard and are bigger and easier to click.
 
-Make a link the minimum phrase that is enough to understand what will happen without reading the whole text.
+<table style="none" border="false">
+  <tr>
+    <td>
+      <format color="369650" style="bold">Correct</format>
+      <img src="link_when_not_to_2_correct.png" width="706" alt="A stack of controls with a button 'Edit' following one of the controls."/>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <format color="E55765" style="bold">Incorrect</format>
+      <img src="link_when_not_to_2_incorrect.png" width="706" alt="A stack of controls with a link 'Edit' following one of the controls."/>
+    </td>
+  </tr>
+</table>
 
-![](09_part_of_text_2.png){width=489}
+## How to use
 
-### Link in a tree or table
+### Writing guidelines
 
-Use a link as a secondary action for some items of a tree or table.
+#### Use sentence capitalization
 
-![](10_tree.png){width=387}
+Follow the [sentence capilaziation](capitalization.md#sentence) guideline.
 
-If an action is needed for all items of a tree or table:
+<table style="none" border="false">
+  <tr>
+    <td>
+      <format color="369650" style="bold">Correct</format>
+      <img src="link_how_to_1_correct.png" width="378" alt="A 'Modify options' drod-down with the sentence style capitalization."/>
+    </td>
+    <td>
+      <format color="E55765" style="bold">Incorrect</format>
+      <img src="link_how_to_1_incorrect.png" width="378" alt="A 'Modify Options' drod-down with the title style capitalization."/>
+    </td>
+  </tr>
+</table>
 
-* Add an [icon button](icon_button.md) to a toolbar.
-* For a list of choices, add a separate table column of drop-down cells (see [Table](table.md#editing-values)).
-* If a link in every tree or table line has meaningful formatting, leave it:
-  ![](11_table.png){width=342}
-  *Links in the "Push Commits" window have the format that helps understand the relation between them: "[remote repository] : [branch]". Moving these links elsewhere would break this meaning.*
+#### Ellipsis
 
-### External link icon
+Add an ellipsis to a button-link if it opens another UI area where input is possible. See examples for the [Button](button.topic).
```

Comment:
The phrase "button-link" is used two times in the article, let's not use it at all. Here it could be just link